server/libs/infra/src/db/entities/album.entity.ts

```diff
@@ -35,11 +36,13 @@ export class AlbumEntity {
   @Column({ comment: 'Asset ID to be used as thumbnail', type: 'varchar', nullable: true })
   albumThumbnailAssetId!: string | null;
 
-  @OneToMany(() => UserAlbumEntity, (userAlbums) => userAlbums.albumInfo)
-  sharedUsers?: UserAlbumEntity[];
+  @ManyToMany(() => UserEntity, { eager: true })
+  @JoinTable()
+  sharedUsers!: UserEntity[];
 
-  @OneToMany(() => AssetAlbumEntity, (assetAlbumEntity) => assetAlbumEntity.albumInfo)
-  assets?: AssetAlbumEntity[];
+  @ManyToMany(() => AssetEntity, { eager: true })
+  @JoinTable()
+  assets!: AssetEntity[];
```

Comment:
This is current behavior, so it's fine, but might have to re-visit it. Not sure what performance impact is for loading all of the assets for all of the albums from the database on a large (album-heavy) library.